.dockerignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+build
```

Comment:
please add some comments to explain it
see the following example:
https://gist.github.com/jkinkead/55db75d57df756c6d66b4c3f395df521